dom/package.json

```diff
@@ -33,6 +33,19 @@
   "module": "lib/es6/index.js",
   "typings": "lib/cjs/index.d.ts",
   "types": "lib/cjs/index.d.ts",
+  "scripts": {
+    "build": "pnpm run build:cjs && pnpm run build:es6",
+    "build:cjs": "tsc --module commonjs --outDir ./lib/cjs",
+    "build:es6": "tsc --module es6 --outDir ./lib/es6",
```

Comment:
Minor nitpick but it would be more consistent to write `build-es6` since we use hyphens in other scripts